src/components/Header/styles.js

```diff
@@ -1,30 +1,54 @@
 import { withStyles } from '@material-ui/core';
 
 
-const styles = {
+const styles = theme => ({
   root: {
     backgroundColor: 'white',
+    minHeight: 61,
   },
 
   content: {
     maxWidth: '960px',
     width: '100%',
     margin: '0 auto',
+    display: 'flex',
+    justifyContent: 'center',
+
+    [theme.breakpoints.up('sm')]: {
+      justifyContent: 'space-between',
+    }
   },
 
   button: {
-    width: '100%',
-    display: 'flex',
-    justifyContent: 'flex-end',
+    display: 'none',
+
+    [theme.breakpoints.up('sm')]: {
+      width: '100%',
+      height: 46,
+      display: 'flex',
+      justifyContent: 'flex-end',
+    }
   },
 
   buttonText: {
     fontWeight: 700,
     padding: '4px 13px',
     fontSize: '1.1rem',
     fontFamily: 'Oswald, Arial, sans-serif',
+  },
+
+  buttonLeft: {
+    justifyContent: 'flex-start',
+    padding: 0,
+    '&:hover': {
+      backgroundColor: 'transparent',
+    },
+  },
+
+  buttonLogo: {
+    maxWidth: 192
```

Comment:
Are these pixels? If possible I think we should start moving to a logarithmic rem system. Also not urgent, we can do this when we get back to this.

Reply:
Yes, it is pixels. I'll make a note for this.